docker/images/dev/Dockerfile

```diff
@@ -1,7 +1,12 @@
 # The base image we inherit from is richie:latest, but you can override this by
-# passing a build argument to your build command, e.g.:
+# passing the BASE_TAG build argument to your build command. You can also
+# override the default container running user ID (e.g. 1000) thanks to the UID
+# build argument (we recommend using yours). An example follows:
```

Comment:
Should be called USER (or USER_ID)? Not sure UID is explicit and the param in Dockerfile is called USER.

Reply:
It refers to the UID shell variable. I thought it was explicit...

Reply:
ok then